### DISCUSSION TITLE
Plans to cut a new release?

### DISCUSSION THREAD

**Discussion by noodnik2** [cucumber/godog#655] [Q&A] [answered]
Hi, 

Are there plans or thoughts to cut a new release in the near future?  My team would be interested in leveraging several useful features that were recently merged into the `main` branch.

-- Marty

**Comment by vearutop** [comment, marked as answer]
Yes, I think it is a good idea, I'll prepare changelog for a new release and tag it a bit later.

**Reply by vearutop** [reply]
`v0.15.0` tagged, sorry for delay.
